src/Sarif/ConsoleLogger.cs

```diff
@@ -320,7 +320,9 @@ private string FormatNotificationMessage(Notification notification)
                 throw new InvalidOperationException("Unknown notification level: " + notification.Level);
             }
 
-            var sb = new StringBuilder(issueType + " ");
+            var sb = new StringBuilder(_toolName + " : ");
```

Comment:
>_toolName [](start = 39, length = 9)

This change is required in order for the MSBuildConverter to transform a notification into a recorded result. Currently, the converter will treat this as a tool result, not a notification. My proposal is that this specific code path, i.e., matching the tool name rather than a file name, should produce a tool configuration notification. That would be a change in the converter itself.